src/task.h

```diff
@@ -199,19 +196,35 @@ task_enqueue(task_system *ts,
              bool         at_head);
 
 /*
- * Performs one task if there is one waiting.  Otherwise returns
- * immediately.  Returns STATUS_TIMEDOUT to indicate that there was no
- * task to run.
+ * Possibly performs one background task if there is one waiting,
+ * based on the specified queue_scale_percent.  Otherwise returns
+ * immediately.  Returns STATUS_TIMEDOUT to indicate that it did not
+ * run any task.  STATUS_OK indicates that it did run a task.
+ * queue_scale_percent specifies how big the queue must be, relative
+ * to the number of background threads for that task group, for us to
+ * perform a task in that group.  So, for example,
+ * - A queue_scale_percent of 0 means always perform a task if one is
+     waiting.
+ * - A queue_scale_percent of 100 means perform a task if there are as
```

Comment:
minor: Should this line read: "... there are at least as many waiting tasks ..." ?